octobot_trading/personal_data/orders/order.py

```diff
@@ -471,6 +477,32 @@ def update_from_raw(self, raw_order):
             reduce_only=raw_order.get(enums.ExchangeConstantsOrderColumns.REDUCE_ONLY.value, False)
         )
 
+    async def update_from_order(self, other_order):
```

Comment:
Why not using `self.update(` with `other_order` attributes ?

Reply:
We could use self.update(), here I think it's better to define a real method that will update all the necessary attributes in one place when we have an existing order instead of making a huge update() for which we will need to specify each attribute every other time we will need to udpate an order from another one.
This way we know that it's only in this method that we can update an order from another order thazt is already in octobot